gemrb/includes/win32def.h

```diff
@@ -32,7 +32,6 @@
 #define UNICODE
 #define _UNICODE
 #define NOGDI
-#define NOCOLOR
```

Comment:
There are a few other NOCOLOR mentions in the tree that need to be changed.

Reply:
Any others?